lightning-tag/data-collection.md

```diff
@@ -29,3 +48,51 @@ behaviors | New or existing behaviors as type/value pairs, where the type is a s
 ruleBuilder	| Custom keys and values to be used for the Rule Builder tool within the DMP | data: {<br/>&nbsp;&nbsp;ruleBuilder: {<br/>&nbsp;&nbsp;&nbsp;&nbsp;article_tags: ['food', 'in the news'],<br/>&nbsp;&nbsp;&nbsp;&nbsp;article_title: ['Todays Headline'],<br/>&nbsp;&nbsp;&nbsp;&nbsp;article_author: ['Bob Roberts'],<br/>&nbsp;&nbsp;},<br/>},
 thirdParty | An identifier to associate with the current browser, typically to enable server side data transfer. Your Lotame representative will provide the namespace value as necessary for your implementation. | data: {<br/>&nbsp;&nbsp;thirdParty: {<br/>&nbsp;&nbsp;&nbsp;&nbsp;namespace: 'FAKE',<br/>&nbsp;&nbsp;&nbsp;&nbsp;value: '123456789101112131415'<br/>&nbsp;&nbsp;},<br/>},
 sha256email	| The current users email address, first lower-cased, trimmed of whitespace, then hashed using SHA256	|data: {<br/>&nbsp;&nbsp;sha256email: 'lowercase_no_whitespace_sha256_hashed_email'<br/>},
+
+## Passing Data outside of Page Load
+
+### Page Interactions
+
+There are times when a Lotame Client wants to pass data to Lotame based on interactions with a page that you customer makes after the page load. For example, interactions with a video player. To do this, you can use the Lightning Tag `collect()` method to send that data even after the config data collection happens as described above.
+
+This option takes the same `{data}` object as describe at the top of the page.
+
+```javascript
+window.lotame_<lotameClientId>.collect({
+  behaviorIds: [CSV_OF_BEHAVIOR_IDS],
+  behaviors: {
+  int: ['behaviorName', 'behaviorName2'],
+  act: ['behaviorName']
+  },
+  ruleBuilder: {
+  key1: ['value 1a', 'value 1b']
+  },
+  thirdParty: {
+  namespace: 'NAMESPACE',
+  value: 'TPID_VALUE'
+  },
+  sha256email: 'SHA256_VALUE'
+});
+```
+
+### New Page Load Events
+
+Lightning Tag provides a `page()` method that you can pass data on just like the `collect()` method described above. Unlike the `collect()` method, the `page()` method records a pageView in Lotame and initiates targeting again which will update audiences in your audience Callback or LocalStorage. An example use-case would be an infinite scrolling site that loads new "articles" once the current article is scrolled to the bottom. `page()` allows your site to note the new article pageView and pass data in that corresponds to the new article so targeting can be rerun based on this new information.
```

Comment:
The main use case I would reference here is a Single Page App where the url remains the same, no page load event occurs, but the main content of the window is replaced. @ctmitchell28 - would optimus handle the above just fine? We actually want them to use this sparingly.

Reply:
Clarified the wording using your suggestion.